.github/workflows/main.yml

```diff
@@ -12,17 +12,10 @@ on:
   release:
     types: [published]
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.ref != 'refs/heads/main' || github.run_number }}
-  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
```

Comment:
This makes sure that we only run pipelines for latest commits to a PR. Why are you removing it?

Reply:
I spoke to @alexzbusko this needs to be tweaked to work with reusable workflow so it's okay to remove for now